scripts/build_proto

```diff
@@ -17,5 +17,5 @@ else
     find src/syft/proto -name "*_pb2.py" -print0 | xargs -0 sed -i 's/from \(proto.*\) import /from syft.\1 import /g'
 fi
 
-isort src/syft/proto/**/*.py
```

Comment:
I'm not sure what this does but i trust you

Reply:
Disabling isort until we configure it properly otherwise it’s going to keep swapping all the sorts around to other people’s changes every time I commit.